extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -366,6 +370,28 @@ protected function disableWikis( $wikis, &$deleted = array(), &$notDeleted = arr
 		}
 	}
 
+	private function disableDiscussion( $cityId ) {
+		try {
+			$this->getSitesApi()->softDeleteSite( $cityId, F::app()->wg->TheSchwartzSecretToken );
+		}
+		catch ( \Swagger\Client\ApiException $e ) {
+			echo "{$cityId} Failed to soft delete Discussion site: {$e->getMessage()}\n";
```

Comment:
Again, maybe we should be using `WikiaLogger::instance()->error()` here instead?